cpp/ql/test/library-tests/dataflow/dataflow-tests/has-parameter-flow-out.expected

```diff
@@ -0,0 +1,2 @@
+failures
```

Comment:
It's a bit of a shame that we also have the `failures` predicate here, but I don't see an easy way to avoid that. Should of course disappear when we have converted all existing tests away from the legacy interface.